adafruit_bitmap_font/pcf.py

```diff
@@ -291,7 +297,7 @@ def _read_properties(self):
             else:
                 yield (string_map[name_offset], value)
 
-    def load_glyphs(self, code_points):
+    def load_glyphs(self, code_points: Union[int, str, Iterable]) -> None:
```

Comment:
I should have said this in the first review, but we can be even more specific here.  `Iterable` is a generic, so we can define what it gives, which is in this case `int`:

```suggestion
    def load_glyphs(self, code_points: Union[int, str, Iterable[int]]) -> None:
```

Reply:
No worries, I will update! :)